wftests/warrior_tests/data/cli_resp_Data.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" ?>
+<credentials>
+
+
+    <system name="sys1">
+                    <ip>168.127.134.184</ip>>
```

Comment:
move the tests under FNC specific testing.

Reply:
Hi Santhosh,
As you suggested, moved the tests to fnc_warrior_tests repo.